src/Extension/ExtendableTrait.php

```diff
@@ -410,16 +410,16 @@ public function extendableCall($name, $params = null)
             $extension = $this->extensionData['methods'][$name];
             $extensionObject = $this->extensionData['extensions'][$extension];
 
-            if (method_exists($extension, $name) && is_callable([$extension, $name])) {
-                return call_user_func_array([$extensionObject, $name], $params);
+            if (method_exists($extension, $name)) {
+                return call_user_func_array([$extensionObject, $name], array_values($params));
             }
         }
 
         if (isset($this->extensionData['dynamicMethods'][$name])) {
             $dynamicCallable = $this->extensionData['dynamicMethods'][$name];
 
             if (is_callable($dynamicCallable)) {
-                return call_user_func_array($dynamicCallable, $params);
+                return call_user_func_array($dynamicCallable, array_values($params));
```

Comment:
Reason for change?

Reply:
In php8 named parameters were introduced and now it is required to match called method parameter name when setting parameters by array destructing or call_user_func_array() etc.